src/components/VideoPlayer/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {FC} from 'react';
+import ReactPlayerYouTube from 'react-player/youtube';
+import ReactPlayerVimeo from 'react-player/vimeo';
+
+import {Source} from 'types/tutorials';
+
+import './VideoPlayer.scss';
+
+interface VideoPlayerProps {
+  controls?: boolean;
+  source: Source;
+  videoId: string;
+}
+
+const VideoPlayer: FC<VideoPlayerProps> = ({controls = true, source, videoId}) => {
+  return (
+    <div className="VideoPlayer">
+      {source === Source.youtube ? (
+        <ReactPlayerYouTube
+          className="VideoPlayer__player"
+          controls={controls}
+          height="100%"
+          url={`https://www.youtube.com/watch?v=${videoId}`}
```

Comment:
Since we're storing only `video_id`, I have to do this. If we stored the complete video URL, it could be passed in directly.

Reply:
interesting, could we have a helper function in utils so that we can do something like:

`url={getVideoUrl(videoId, VideoType.youtube)}`?